src/SelfService/Domain/Services/ECRRepositoryService.cs

```diff
@@ -164,16 +152,15 @@ public async Task SynchronizeAwsECRAndDatabase(bool performUpdateOnMismatch)
         {
             List<ECRRepository> newRepositories = new();
 
-            foreach (var repositoryName in repositoriesNotInDb)
+            foreach (var name in repositoriesNotInDb)
             {
                 newRepositories.Add(
                     new ECRRepository(
                         new ECRRepositoryId(),
-                        repositoryName,
+                        name,
                         CreatedByCloudEngineeringTeamDescription,
-                        repositoryName,
                         _cloudEngineeringTeamUserId,
-                        null
+                        DateTime.UtcNow
```

Comment:
should be null :)

Reply:
Yeah, makes sense after our chat on slack.
Changed